sos/report/plugins/foreman.py

```diff
@@ -178,7 +181,11 @@ def setup(self):
         )
         self.add_cmd_output(_cmd, suggest_filename='foreman_db_tables_sizes',
                             env=self.env)
+        self.collect_foreman_db()
```

Comment:
In this file, pylint was complaining that 'R0912 too-many-branches' and 'R0914 too-many-locals' - essentially the function being too big. So split it into 3.